master/buildbot/test/unit/test_worker_ec2.py

```diff
@@ -190,6 +190,38 @@ def test_start_spot_instance(self):
         self.assertEqual(instances[0].tags, {})
 
     @mock_ec2
+    def test_start_spot_instance_tags(self):
+        c = self.botoSetup()
+        amis = c.get_all_images()
+        tags = {'foo': 'bar'}
+        product_description = 'Linux/Unix'
+        with assertProducesWarnings(
+                DeprecatedWorkerNameWarning,
+                messages_patterns=[
+                    r"Use of default value of 'keypair_name' of "
```

Comment:
this text could use `'` instead of `"`

I wonder why this text got double quotes while the rest got single quotes....